tutorial/deepsparse_server_config.yaml

```diff
@@ -0,0 +1,10 @@
+num_cores: 2
+num_workers: 2
+loggers:
```

Comment:
I guess this will need to be changed after our loggers refactor? Want to get this merged in before all that or after?

Reply:
Afaik, not at all. We have agreed that loggers remain a global dictionary of the config file:
<img width="693" alt="image" src="https://user-images.githubusercontent.com/97082108/195637688-18110cda-169a-4cfb-a9f4-bb3dd7c35a4e.png">